### ISSUE TITLE
Model Spese has data not encrypted

### ISSUE THREAD

**Issue by davideferre** [dennybiasiolli/budgest-server#9]
A possible warning is that if someone get the db, they are able to read all your expenses. My suggest is to use [django-encrypted-fields](https://github.com/defrex/django-encrypted-fields) at least for _importo_ field in _Spese_ model from _budgest.models_.

**Comment by dennybiasiolli** [comment]
@davideferre thanks for the suggestion! 😄

**Comment by dennybiasiolli** [comment]
@davideferre can `importo` field be used in query linke "sum" after encryption? In the documentation I read "_Warning: Once the data is encrypted, it can no longer to used to query or sort. In SQL, these will all look like text fields with random noise in them (which is what you want)._"

**Comment by davideferre** [comment]
No, you have to query without any operations or filters and make this work in python. The field becomes a string after the encryption.
